src/components/availability-label/availability-label.tsx

```diff
@@ -13,79 +14,90 @@ export interface AvailabilityLabelProps {
   handleSelectManifestation?: () => void | undefined;
   cursorPointer?: boolean;
 }
+export const AvailabilityLabel: React.FC<AvailabilityLabelProps> = withConfig(
```

Comment:
🤔 I am still no sure whether this is the right place to add this. Have you tried updating the story to `component: withConfig(AvailabilityLabel)` instead?

If we cannot figure this out then perhaps we could leave this as is, merge it and talk to Mikkel about it on monday.